cpp/cmake/StandardConfig.cmake

```diff
@@ -175,7 +175,10 @@ function(StandardConfig config_type)
 
   if (NOT MSVC)
     # pragmas can be needed for specific optimizations in some compilers while ignored in other
-    add_compile_options(-Wall -Wextra -pedantic -Werror -Wno-unknown-pragmas)
+    add_compile_options(-Wall -Wextra -pedantic)
+    add_compile_options("$<$<COMPILE_LANGUAGE:CXX>:-Werror>")
```

Comment:
@psilar Please make enable those for plain C too (we have at least one purposefully plain-C project- https://github.com/provizio/provizio_radar_api_core). You could also include them in a single expression/per language.

Reply:
Done